spark/src/main/java/io/jaegertracing/analytics/spark/SparkKinesisRunner.java

```diff
@@ -85,27 +100,30 @@ public static void main(String[] args) throws InterruptedException, IOException
         String region = Regions.getCurrentRegion()!=null ? Regions.getCurrentRegion().getName()
                 : getPropOrEnv("AWS_REGION", Regions.US_EAST_1.getName());
 
+        String service_endpoint = getPropOrEnv("KINESIS_ENDPOINT", "https://kinesis.us-east-1.amazonaws.com");
```

Comment:
throw error if not provided. We can't try to read from other region and all right.